### PR TITLE
Replace unscoped with UnscopedRefAttribute

### PR DESCRIPTION
**Pull request by cston**

### CODE REVIEW

File: proposals/low-level-struct-improvements.md [hunk 1]
```diff
@@ -383,9 +381,10 @@ The presence of `scoped` allows developers to reduce the friction this rule crea
 
 Impact of this change is discussed more deeply [below](#examples-method-arguments-must-match). Overall this will allow developers to make call sites more flexible by annotating non-escaping ref-like values with `scoped`.
 
-The `scoped` modifier on parameters also impacts our object overriding, interface implementation and `delegate` conversion rules. The signature for an override, interface implementation or `delegate` conversion can: 
+The `scoped` modifier and `[UnscopedRef]` attribute (see [below](#rules-unscoped)) on parameters also impacts our object overriding, interface implementation and `delegate` conversion rules. The signature for an override, interface implementation or `delegate` conversion can:
```
Comment (jcouv) [inline proposals/low-level-struct-improvements.md:384]: What are the rules for `[UnscopedRef]` in OHI? Is any variance allowed?

File: proposals/low-level-struct-improvements.md [hunk 2]
```diff
@@ -442,7 +441,7 @@ The [rationale](https://github.com/dotnet/csharplang/blob/main/proposals/csharp-
 
 <a name="rules-unscoped"></a>
 
-To fix this the  language will provide the opposite of the `scoped` lifetime annotation in the syntax `unscoped`.  The keyword `unscoped` will be used to expand the lifetime of a value. It can be applied to any `ref` which is implicitly `unscoped` and has the impact of changing its *ref-safe-to-escape* to *calling method*.
+To fix this the  language will provide the opposite of the `scoped` lifetime annotation by supporting an `UnscopedRefAttribute`.  The attribute can be used to expand the lifetime of a value. It can be applied to any `ref` which is implicitly `scoped` and has the impact of changing its *ref-safe-to-escape* to *calling method*.
```
Comment (jcouv) [inline proposals/low-level-struct-improvements.md:444]: >any `ref` which is implicitly `scoped` [](http://example.com/codeflow?start=212&length=38)

Consider enumerating those implicitly `scoped` `ref` cases if it's easy.

**Review by jcouv — COMMENTED** [review]
Done with review pass (iteration 2). Looks good except for a question on OHI. Thanks much!

File: proposals/low-level-struct-improvements.md (hunk 2 above)
Comment (jcouv) [reply proposals/low-level-struct-improvements.md:444]: Also, is it an error to place `[UnscopedRef]` on some location that doesn't qualify per above?

File: proposals/low-level-struct-improvements.md (hunk 1 above)
Comment (cston) [reply proposals/low-level-struct-improvements.md:384]: Thanks. Added.

File: proposals/low-level-struct-improvements.md (hunk 2 above)
Comment (cston) [reply proposals/low-level-struct-improvements.md:444]: > Consider enumerating those implicitly scoped ref cases if it's easy.

Added a link to the cases described earlier.

> Also, is it an error to place `[UnscopedRef]` on some location that doesn't qualify per above?

Added as an error.

File: proposals/low-level-struct-improvements.md [hunk 3]
```diff
@@ -442,7 +443,9 @@ The [rationale](https://github.com/dotnet/csharplang/blob/main/proposals/csharp-
 
 <a name="rules-unscoped"></a>
 
-To fix this the  language will provide the opposite of the `scoped` lifetime annotation in the syntax `unscoped`.  The keyword `unscoped` will be used to expand the lifetime of a value. It can be applied to any `ref` which is implicitly `unscoped` and has the impact of changing its *ref-safe-to-escape* to *calling method*.
+To fix this the  language will provide the opposite of the `scoped` lifetime annotation by supporting an `UnscopedRefAttribute`.
+The attribute can be used to expand the lifetime of a value. It can be applied to any `ref` which is [implicitly `scoped`](#implicitly-scoped) and has the impact of changing its *ref-safe-to-escape* to *calling method*.
+An error is reported if `[UnscopedRef]` is applied to a parameter passed by value or to a `ref` that is not considered scoped.
```
Comment (jcouv) [inline proposals/low-level-struct-improvements.md:448]: >not considered scoped. [](http://example.com/codeflow?start=104&length=22)

"not considered implicitly scoped"? That will cover the `[UnscopedRef] scoped` scenario as well (line 500). Can this line (448) be moved to line 500 for various errors reported on misused `[UnscopedRef]`?

**Review by jcouv — APPROVED** [review]
LGTM Thanks (iteration 4)

**Pull request merged**